PokeAlarm/WebhookStructs.py

```diff
@@ -52,6 +52,10 @@ def pokemon(data):
             'id': data['encounter_id'],
             'pkmn_id': int(data['pokemon_id']),
             'disappear_time': datetime.utcfromtimestamp(data['disappear_time']),
+            'time_until_despawn': data['seconds_until_despawn'],
+            'spawn_start': data['spawn_start'],
```

Comment:
Does monocle have these fields as well?

Reply:
I will set up a monocle instance to verify

Reply:
If it doesn't, we'll need to switch these to use `.get()` and default to `unknown`